### DISCUSSION TITLE
Kinematic Error information

### DISCUSSION THREAD

**Discussion by hisashiishida** [surgical-robotics-ai/surgical_robotics_challenge#13] [Q&A] [answered]
When and how will be getting the information about the kinematic error?

**Comment by adnanmunawar** [comment, marked as answer]
Thanks for the question Hisashi. We just added the option to add joint errors in this commit d4ff9ea75bd0b4100b15f1f8fb2399ee1181a13f. Stay tuned for information regarding what joints would the errors be added to and what is the maximum range of the error.
